src/main/java/com/cleanroommc/groovyscript/gameobjects/GohMetaMethod.java

```diff
@@ -0,0 +1,56 @@
+package com.cleanroommc.groovyscript.gameobjects;
+
+import com.cleanroommc.groovyscript.mapper.ObjectMapper;
+import com.cleanroommc.groovyscript.sandbox.expand.IDocumented;
+import groovy.lang.MetaMethod;
+import org.codehaus.groovy.reflection.CachedClass;
+import org.codehaus.groovy.reflection.ReflectionCache;
+import org.jetbrains.annotations.ApiStatus;
+
+import java.lang.reflect.Modifier;
+
+@Deprecated
+@ApiStatus.ScheduledForRemoval(inVersion = "1.2.0")
+public class GohMetaMethod extends MetaMethod implements IDocumented {
```

Comment:
this file is introduced in this PR in its current deprecated state. i dont see anywhere `GohMetaMethod` is used, either.